CONTRIBUTING.md

```diff
@@ -22,3 +22,28 @@ Run `yarn lint --fix` to automatically format files.
 If you can, take look at [commands.md](commands.md) and check out [Wadler's paper](http://homepages.inf.ed.ac.uk/wadler/papers/prettier/prettier.pdf) to understand how Prettier works.
 
 If you want to know more about Prettier's GitHub labels, see the [Issue Labels](https://github.com/prettier/prettier/wiki/Issue-Labels) page on the Wiki.
+
+## Performance
+
+If you're contributing a performance improvement, the following Prettier CLI options can help:
+
+- `--debug-benchmark` uses [`benchmark`](https://npm.im/benchmark) module to produce statistically significant time measurements. The measurements are printed at the debug log level, use `--loglevel debug` to see them.
+- `--debug-repeat N` uses a naive loop to repeat the formatting `N` times and measures the average run time. It can be useful to highlight hot functions in the profiler. The measurements are printed at the debug log level, use `--loglevel debug` to see them.
+
+To run Prettier with the profiler, use the Node CLI `--inspect-brk` option and Chromium/Chrome browser's [`chrome://inspect`](chrome://inspect) page to activate the Chrome DevTools Profiler for Node:
+
+```
+$ cat FileToFormat.js | NODE_ENV=production node --inspect-brk ./bin/prettier.js --stdin-filepath FileToFormat.js --debug-repeat 1000 --loglevel debug > /dev/null
```

Comment:
We might want to recommend building it and using `./dist/bin-prettier.js` instead of `./bin/prettier.js` to get more accurate profiles (e.g. because of these `process.env.NODE_ENV` accesses that slow down `concat`).